js/constructor.js

```diff
@@ -11,6 +13,8 @@ function Store(id, name, minCust, maxCust, avgCookieCust){
     this.cookieSoldDay = [];
     this.cookieDayTotal = 0;
     patStores.push(this);
+    this.custHour();
+    this.cookieSoldHour();
     this.render();
 }
 Store.prototype.custHour = function(){
```

Comment:
Instead of having an extra variable `this.storeHours` you could just use `hours.length`. Just to remove the possibility of a bug :)